Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "mozjs_sys"
 description = "System crate for the Mozilla SpiderMonkey JavaScript engine."
 repository = "https://github.com/servo/mozjs/"
-version = "0.67.1"
+version = "0.68.0"
```

Comment:
Should this be 0.72.0?

Reply:
No. We shouldn't be following that version because then we end up in trouble when we want to introduce additional breaking changes.

Reply:
FWIW, when I did the smup, I'd set NEW_VERSION to be max(OLD_VERSION+1, SPIDERMONKEY_VERSION).

Reply:
With this scheme you can end up with a `0.74` that is actually from Firefox 72, and that's just too confusing to me, so let's just use version numbers as they were intended to be used.